silver/utils/decorators.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2016 Presslabs SRL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from django.core.cache import cache
+
+
+def cached_method(unique_attribute=None):
+    """
+    A decorator for caching method results in the Django context
+    :param unique_attribute: Name of attribute used in the cache key
+    """
+    def decorator(func):
+        def func_wrapper(self, *args, **kwargs):
```

Comment:
`self` may be misleading because is used to refer to the current class instance. Here you are not working in the interior of a class, but in a function. Maybe `obj` is better, of `model`.